# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fuse/no-dev-kvm.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Created file outside allowed top-level dir: dev/fuse

--- libfuse-fuse-3.18.1/util/install_helper.sh.vanilla	2026-01-19 17:17:39.580991013 +0100
+++ libfuse-fuse-3.18.1/util/install_helper.sh	2026-01-19 17:18:42.929013437 +0100
@@ -31,7 +31,7 @@
     chown root:root "${DESTDIR}${bindir}/fusermount3"
     chmod u+s "${DESTDIR}${bindir}/fusermount3"
 
-    if test ! -e "${DESTDIR}/dev/fuse"; then
+    if false; then
         mkdir -p "${DESTDIR}/dev"
         mknod "${DESTDIR}/dev/fuse" -m 0666 c 10 229
     fi
